data/human/weapons.txt

```diff
@@ -11,6 +11,194 @@
 # You should have received a copy of the GNU General Public License along with
 # this program. If not, see <https://www.gnu.org/licenses/>.
 
+outfit "Tethys Cannon"
+	category "Turrets"
+	cost 1067000
+	mass 128
+	thumbnail "outfit/rnd_gun2_turret"
+	"outfit space" -110
+	"weapon capacity" -97
+	"drag" 7
+	"required crew" 4
+	"turret mounts" -1
+	"shell capacity" 40
+	"active cooling" 4
+	"cooling energy" 3
+	weapon
+		safe
+		"hardpoint sprite" "hardpoint/rnd_gun2_turret_tethys"
```

Comment:
This should be renamed to fit more in line with our naming scheme. Something like "tethys turret". The image files should be renamed as well.
(The same applies to the icons and outfit images as well)